notebooks/MAST/HSC/CASJOBS/SWEEPS_HSCV3P1_API/sweeps_hscv3p1_api.ipynb

```diff
@@ -0,0 +1,969 @@
+    "# the default pagesize is 50000 rows\n",
+    "\n",
+    "t0 = time.time()\n",
+    "results = hscsearch(table=\"propermotions\", release='v3', columns=columns, verbose=True, pagesize=500000, **constraints)\n",
+    "print(\"{:.1f} s: retrieved data\".format(time.time()-t0))\n",
+    "tab = ascii.read(results)\n",
+    "print(f\"{(time.time()-t0):.1f} s: converted to astropy table\")\n",
+    "\n",
+    "# change some column names for consistency with the Casjobs version of this notebook\n",
+    "tab.rename_column(\"raMean\", \"RA\")\n",
+    "tab.rename_column(\"decMean\", \"Dec\")\n",
+    "tab.rename_column(\"raMeanErr\", \"RAerr\")\n",
+    "tab.rename_column(\"decMeanErr\", \"Decerr\")\n",
+    "tab.rename_column(\"pmLat\", \"bpm\")\n",
+    "tab.rename_column(\"pmLon\", \"lpm\")\n",
+    "tab.rename_column(\"pmLatErr\", \"bpmerr\")\n",
+    "tab.rename_column(\"pmLonErr\", \"lpmerr\")\n",
+    "\n",
+    "# compute some additional columns\n",
+    "\n",
+    "tab['pmdev'] = np.sqrt(tab['pmLonDev']**2+tab['pmLatDev']**2)\n",
+    "tab['yr'] = (tab['epochMean'] - 47892)/365.25+1990\n",
+    "tab['dT'] = (tab['epochEnd']-tab['epochStart'])/365.25\n",
+    "tab['yrStart'] = (tab['epochStart'] - 47892)/365.25+1990\n",
+    "tab['yrEnd'] = (tab['epochEnd'] - 47892)/365.25+1990\n",
+    "\n",
+    "# delete some columns that are not needed after the computations\n",
+    "del tab['pmLonDev'], tab['pmLatDev'], tab['epochEnd'], tab['epochStart'], tab['epochMean']\n",
+    "\n",
+    "tab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Properties of Full Catalog  <a class=\"anchor\" id=\"fullcat\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sky Coverage  <a class=\"anchor\" id=\"SkyCoverage\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x = tab['RA']\n",
+    "y = tab['Dec']\n",
+    "\n",
+    "plt.figure(1, (12, 10))\n",
+    "plt.scatter(x, y, s=1)\n",
+    "plt.autoscale(tight=True)\n",
+    "plt.xlabel('RA')\n",
+    "plt.ylabel('Dec')\n",
+    "dc = 0.01\n",
+    "plt.xlim(min(x)-dc, max(x)+dc)\n",
+    "plt.ylim(min(y)-dc, max(y)+dc)\n",
+    "plt.gca().invert_xaxis()\n",
+    "plt.text(0.5, 0.93, f'{len(x):,} stars in SWEEPS', horizontalalignment='left', transform=plt.gca().transAxes)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Proper Motion Histograms <a class=\"anchor\" id=\"pmhist\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Proper motion histograms for lon and lat "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bin = 0.2\n",
+    "hrange = (-20, 20)\n",
+    "bincount = int((hrange[1]-hrange[0])/bin + 0.5) + 1\n",
+    "plt.figure(1, (12, 10))\n",
```

Comment:
Missing space? (and similar in following cells)

Reply:
@eteq What do you mean missing space?

Reply:
done